### ISSUE TITLE
Max Object Count and VaRest

### ISSUE THREAD

**Issue by WSPluta** [ufna/VaRest#395]
I'm using varest to pull data and update status of 1000 meshes via ID.

When I tried to compile to for Oculus I keep getting max objects count going very high. So I introduced collect garbage evey 50 rows read

But Ideally I want for it to read all 1000 nodes in real time, but cant understand why my object count goes so high?

**Comment by ufna** [comment]
Please check `gc.MaxObjectsInGame`, it's somewhy very low on Android by default.

VaRest uses UObjects internally to give access to Blueprints.

**Comment by WSPluta** [comment]
yeah i tried that but because in total im getting    VaRestJsonValue  2,421,102 its way over
I have to read REST in sequence and collect garbage every 50 rows which is not ideal
